src/test/java/com/snowflake/kafka/connector/ConnectClusterBaseIT.java

```diff
@@ -94,7 +98,7 @@ protected final void waitForConnectorStopped(String connectorName) {
     try {
       connectCluster
           .assertions()
-          .assertConnectorAndTasksAreStopped(connectorName, "Failed to stop the connector");
+          .assertConnectorIsStopped(connectorName, "Failed to stop the connector");
```

Comment:
This is more or less the same as `assertConnectorAndTasksAreStopped`. Only name has changed.